storage/src/main/java/fk/prof/storage/buffer/StorageBackedOutputStream.java

```diff
@@ -107,6 +107,8 @@ private void storeAndSwapBuffer() throws IOException {
         try {
             buf = null; // get rid of the reference, in case the borrow fails
             try (Timer.Context context = tmrBuffPoolBorrow.time()) {
+                LOGGER.debug("acquiring buffer for file: {}", fileNameStrategy.getFileName(0));
```

Comment:
These 2 should be 1 line. If we make it 2 lines, they can get fragmented, which is harder to read. Since they are in the same callsite, they can be folded into a single message.